search-java-getting-started/src/main/main/java/azure/search/sample/App.java

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azure.search.sample;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.LocalDateTime;
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.Context;
+import com.azure.search.documents.SearchClient;
+import com.azure.search.documents.SearchClientBuilder;
+import com.azure.search.documents.models.SearchOptions;
+import com.azure.search.documents.indexes.SearchIndexClient;
+import com.azure.search.documents.indexes.SearchIndexClientBuilder;
+import com.azure.search.documents.indexes.models.IndexDocumentsBatch;
+import com.azure.search.documents.indexes.models.SearchIndex;
+import com.azure.search.documents.indexes.models.SearchSuggester;
+import com.azure.search.documents.util.AutocompletePagedIterable;
+import com.azure.search.documents.util.SearchPagedIterable;
+
+public class App {
+
+    public static void main(String[] args) {
+        var searchServiceEndpoint = "https://<your-service>search.windows.net";
```

Comment:
missing a dot

Reply:
Also, do we need a comment here??  Something like 

// Connection to your search service.
// Provide your service name and a valid admin API key.